gtkm/stats/get_stats/aggregate.py

```diff
@@ -0,0 +1,52 @@
+from ...common import gen_url, get_endpoint_data
+
+import json
+import os
+
+
+class GithubAggregateBasicData():
+
+    gtkm_cookie = None
+
+    def __init__(self, gtkm_cookie):
+        self.gtkm_cookie = gtkm_cookie
+
+    async def execute_collecting(self):
+
+        if self.gtkm_cookie:
+            general_user_data = await get_endpoint_data(
+                gen_url('/github/stats/general_user'), cookie=self.gtkm_cookie)
+
+        return general_user_data.json()
+
+
+class GithubAggregateLanguageData():
+
+    gtkm_cookie = None
+
+    def __init__(self, gtkm_cookie):
+        self.gtkm_cookie = gtkm_cookie
+
+    async def execute_collecting(self):
+
+        if self.gtkm_cookie:
+            general_user_data = await get_endpoint_data(
+                gen_url('/github/stats/languages'), cookie=self.gtkm_cookie)
+
+        return general_user_data.json()
+
+
+class GithubAggregateTopRepos():
```

Comment:
Te klasy są mocno powtarzalne.

Można je napisać lepiej.

Stwórz jedną klasę bazową i z niej klasy pochodne, które będą miały np inną wartość ścieżki

np
```python
class GithubAggregateLanguageData(SimpleAggregator):

    gtkm_cookie = None
    ENDPOINT=`github/stats/language`
```

Reply:
Co mogłem, to dałem do klas bazowych. Dodałem nowy plik z basem klasy dla klas agregatora danych. Jakoś bardzo obecnie osobny plik na klasę bazową agragatora nie jest potrzebny, ale jeżeli będziemy rozwijać dalej aplikację, to taki podział zapewni nam większy porządek w kodzie.